### DISCUSSION TITLE
想請問如何自訂義MessageBox的button文字 , 而不使用資源檔的翻譯 ?

### DISCUSSION THREAD

**Discussion by cliffliu123** [irihitech/Ursa.Avalonia#211] [Q&A]
如圖所表示
我想以目前既有的方法去寫MessageBox的button自設文字 而不使用目前您的資源檔

感謝
![image](https://github.com/irihitech/Ursa.Avalonia/assets/6345353/6568000a-cb78-4a23-aa3f-fef03f7e5144)

**Comment by rabbitism** [comment]
两种解决方案
1. 如果想要全局修改这些问题，那么可以直接在引入Semi主题后重新在App ResourceDictionary中重新定义这些动态资源 https://github.com/irihitech/Ursa.Avalonia/blob/f8a11b0f3eb4595b2d65e5791e6ea7f63c5519d4/src/Ursa.Themes.Semi/Locale/zh-cn.axaml#L8.
   如果你觉得需要增加一个新的CultureInfo，欢迎提PR
3. 如果只是针对某一次调用修改，那么message是没有办法实现的，需要自己创建custom dialog

**Comment by cliffliu123** [comment]
我就用方法2好了 , 感謝.
